drivers/storage/portworx/portworx.go

```diff
@@ -53,7 +53,8 @@ const (
 )
 
 var (
-	ErrNodeListEmpty = stderr.New("no nodes were available for px installation")
+	ErrNodeListEmpty  = stderr.New("no nodes were available for px installation")
+	installSecContext = true
```

Comment:
Is the assumption that this will run every time the operator starts? Since it will be set to true on every fresh start

Reply:
Yes,  that is ok... This I added was more for the reconcile loop which calls this function every 20 sec until pre-flight is complete.   However I need to have it executed here and its ok, i believe if it is executed multiple time but figured i would add this to reduce the number of redundant calls while pre-flight was running.